eisbuk-admin/src/store/selectors/slots.ts

```diff
@@ -12,58 +12,58 @@ import { LocalStore } from "@/types/store";
  * @param startDate start of timeframe
  * @returns created selector
  */
-export const getSlotsForCustomer = (
-  category: Category,
-  timeframe: "week" | "month",
-  startDate: DateTime
+export const getSlotsForCustomer =
+  (category: Category, timeframe: "week" | "month", startDate: DateTime) =>
   // eslint-disable-next-line consistent-return
-) => (state: LocalStore): SlotsByDay => {
-  const allSlotsInStore = state.firestore.data?.slotsByDay;
-
-  // return early if no slots in store
-  if (!allSlotsInStore) return {};
-
-  switch (timeframe) {
-    case "month":
-      // get slots for current month
-      const monthString = startDate.toISO().substr(0, 7);
-      const slotsForAMonth = allSlotsInStore[monthString] || {};
-
-      // filter slots from each day with respect to category
-      return Object.keys(slotsForAMonth).reduce((acc, date) => {
-        const [filteredSlotsDay, isFilteredDayEmpty] = filterSlotsByCategory(
-          slotsForAMonth[date],
-          category
-        );
-
-        return !isFilteredDayEmpty ? { ...acc, [date]: filteredSlotsDay } : acc;
-      }, {} as SlotsByDay);
-
-    case "week":
-      // get all week dates
-      const weekDates = Array(7)
-        .fill(startDate)
-        .map((startDate, i) => luxon2ISODate(startDate.plus({ days: i })));
-
-      // filter slots within each day with respect to category
-      const filteredDays = weekDates.reduce((acc, date) => {
-        const monthStaring = date.substr(0, 7);
-        const slotsMonth = allSlotsInStore[monthStaring] || {};
-        const slotsDay = slotsMonth[date] || {};
-
-        const slotKeys = Object.keys(slotsDay);
-        const newDay = slotKeys.reduce((acc, slotId) => {
-          const { categories } = slotsDay[slotId];
-          return categories.includes(category)
-            ? { ...acc, [slotId]: slotsDay[slotId] }
+  (state: LocalStore): SlotsByDay => {
+    const allSlotsInStore = state.firestore.data?.slotsByDay;
+
+    // return early if no slots in store
```

Comment:
I think our editors are playing tug of wars with 2 spaces/4 spaces indentation.
It gets in the way of reviews, so it would be nice to find out why. A first step should be settle on what we want. My vote goes with my editor (2 spaces).